### DISCUSSION TITLE
cluster opeartor down effects

### DISCUSSION THREAD

**Discussion by 282ori** [strimzi/strimzi-kafka-operator#6563] [Q&A]
Hi,
I would like to know about the effection on Kafka when the Strimzi Cluster operator is down.
I know that if the cluster operator is down:
1. New Kafka cluster cannot be created
2. changes on the Kafka CRD's will not be applied

There might be any effect on existing Kafka clusters? 
Will they function us usual?
thanks

**Comment by scholzj** [comment]
The effect is that the cluster is not _operated_. The general expectation is that the operator runs all the time and is managing the cluster. When the cluster operator restarts or when you stop it for few minutes nothing serious should happen. But you should not stop it for days or weeks.
